integration/src/main/java/ai/djl/integration/util/DebugEnvironment.java

```diff
@@ -12,24 +12,19 @@
  */
 package ai.djl.integration.util;
 
-import ai.djl.Device;
 import ai.djl.engine.Engine;
-import ai.djl.util.cuda.CudaUtils;
 import java.io.IOException;
 import java.lang.management.ManagementFactory;
 import java.lang.management.MemoryMXBean;
 import java.lang.management.MemoryUsage;
 import java.nio.ByteOrder;
 import java.nio.charset.StandardCharsets;
 import java.util.Scanner;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 /** Emits debug information about the user environment. */
+@SuppressWarnings("PMD.SystemPrintln")
 public final class DebugEnvironment {
 
-    private static final Logger logger = LoggerFactory.getLogger(DebugEnvironment.class);
```

Comment:
Why do we remove logger and use System.print instead

Reply:
I saw many users mis-configured their logging system and cannot see any logs.